### PR TITLE
Adding _locales submodule pointing to https://github.com/mozilla-l10n/firefox-translations-l10n

### LINKED ISSUE #21: Implement Localization utilizing pontoon

### PR DESCRIPTION
**Pull request by andrenatal**
Fixes #21 

This is inspired on the same approach that https://github.com/mozilla-l10n/multi-account-containers-l10n/ uses. 

I like that because then we won't need actions or bots to push PRs to keep the `_locales` folder synced, and we can just utilize npm runners when we need it updated to both develop as to create builds both manually as on CI.

### CODE REVIEW

File: package.json
Comment (andrenatal) [inline package.json:11]: we need this because `web-ext` fails to load the extension when there's an extraneous folder inside `_locales`

**Review by flodolo — APPROVED** [review]
If this is the approach, we can remove the CODEOWNERS file.

The downside is developing new features or changes that require changes to the en-US file (you'll need a PR to land them in the l10n repository before merging them).

**Comment by andrenatal** [comment]
> If this is the approach, we can remove the CODEOWNERS file.
> 

cool, just removed it 

> The downside is developing new features or changes that require changes to the en-US file (you'll need a PR to land them in the l10n repository before merging them).

we can live with that, don't think there will be too many changes on that

**Pull request merged**
